opennms-webapp/src/main/webapp/WEB-INF/jsp/support/index.jsp

```diff
@@ -185,11 +205,37 @@ session="true"
   <div class="panel-body">
     <ul class="list-unstyled">
       <li><a href="support/about.jsp">About the OpenNMS Web Console</a></li>
-      <li><a href="http://www.opennms.org/documentation/ReleaseNotesStable.html#whats-new">Release Notes</a></li>
-      <li><a href="http://www.opennms.org/wiki/">Online Documentation</a></li>
+      <li><a href="http://docs.opennms.org/opennms/releases/latest/releasenotes/releasenotes.html">Release Notes</a></li>
     </ul>
   </div>
   </div>
+
+  <div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Documentation</h3>
+  </div>
+  <div class="panel-body">
+    <span id="online-documentation" style="display:none;">
+    <ul class="list-unstyled">
+      <li><a href="http://docs.opennms.org/opennms/releases/latest/guide-install/guide-install.html">Installation Guide</a></li>
+      <li><a href="http://docs.opennms.org/opennms/releases/latest/guide-user/guide-user.html">Users Guide</a></li>
+      <li><a href="http://docs.opennms.org/opennms/releases/latest/guide-admin/guide-admin.html">Administrators Guide</a></li>
+      <li><a href="http://docs.opennms.org/opennms/releases/latest/guide-development/guide-development.html">Developers Guide</a></li>
```

Comment:
The only think I've seen the links to the online documentation goes to "latest" instead of the specific version you have running. It can lead to features and functions which don't exist in the version you've running. We don't have a way to ask for a version number right now. Just a quick idea, we could create just one link to the online docs which goes to http://docs.opennms.org/opennms/releases/ and the user has to pick the version he had installed by himself.